frontend/src/components/legal/index.tsx

```diff
@@ -1,11 +1,17 @@
 import * as React from 'react';
 
+import Nav from '../header/nav/index';
+import LegalContent from './legal-content/index';
+import Footer from '../footer/index';
+
 class Legal extends React.Component {
   render() {
     return (
-      <p> 
-        This site is very legal
-      </p>
+      <React.Fragment>
+        <Nav />
```

Comment:
@lustoykov I think Nav should be somehow styled depending on page. 

How would you do this?